### PR TITLE
practice questions

### PR DESCRIPTION
**Pull request by pramodmg**
1. Trying some interview questions about setTimeOut (try.js).
2. Bracket Matching problem (isValidBracket.js)
3. GCD (gcd.js)

### CODE REVIEW

File: javascript/leetcode/try.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The index of this number isss: 1
+4
+The index of this number isss: 0 */
+
+var text = "outside";
+function logIt() {
+  console.log(text);
+  var text = "inside";
+}
+logIt();
```
Comment (SrSandeepKumar) [inline javascript/leetcode/try.js]: What are we checking here?

File: javascript/leetcode/try.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    3000
+  );
+}
+
+/* output: 
+The index of this number is: 0
+The index of this number is: 0
+The index of this number is: 1
+The index of this number is: 2
+The index of this number is: 3 */
```
Comment (SrSandeepKumar) [inline javascript/leetcode/try.js]: This is good one

File: javascript/leetcode/try.js (hunk 1 above)
Comment (pramodmg) [reply javascript/leetcode/try.js]: just some scoping thing ...

**Comment by pramodmg** [comment]
screenshot 2019-03-05 at 9 49 11 pm

using hooks able to achieve a Todo Application.

**Pull request merged**